config/pipeline.yaml

```diff
@@ -983,6 +998,51 @@ build_variants:
 
 build_configs:
 
+  broonie-misc:
```

Comment:
Monitor service will pick up changes in these trees and create events for them but they won't be processed further. 

Should this commit also include changes to `scheduler` section (adding kbuild jobs schedules)?

Reply:
I am adding it step by step, observing possible errors and adding validator logic for such missing changes, like scheduler section.

Reply:
Btw there is enough default kbuild entries in scheduler that will trigger builds on any trees except android.
https://staging.kernelci.org:9000/viewer?search=parent%3D667d30c13e82bd70839e9584

Reply:
Right, all that include [`kbuild-job`](https://github.com/kernelci/kernelci-pipeline/blob/9c96cd9ef70ec0decbd6dd96e71e3510d295b5e7/config/pipeline.yaml#L32) anchor.